flowetl/Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN USER=airflow && \
 RUN mv /entrypoint.sh /defaultentrypoint.sh
 COPY ./entrypoint.sh /entrypoint.sh
 
+WORKDIR /libs
```

Comment:
No need to set the workdir since you're using absolute paths in the next two lines, and then change the WORKDIR again below?